packages/big-design/src/components/Worksheet/editors/ModalEditor/styled.ts

```diff
@@ -0,0 +1,11 @@
+import { theme as defaultTheme } from '@bigcommerce/big-design-theme';
+import styled from 'styled-components';
+
+import { Link } from '../../../Link';
+
+export const StyledLink = styled(Link)`
```

Comment:
Sorry, just caught this. But we probably don't want a link here since it breaks focusability if we don't have an `href`. We probably should use `StyleableButton` instead and make sure keyboard events work properly for this cell.

Reply:
Ok, good point, let me look into it